misc/logs/parse-logs

```diff
@@ -1101,7 +1101,8 @@ for my $rec (@records) {
                         my $file = $spec->[1];
 
                         if (defined $file) {
-                            if (defined $f->{file} && $f->{file} eq $file) {
+                            if (defined $f->{file}
+                                && ($f->{file} eq $file || $module eq 'lua-resty-core' && "stream/$f->{file}" eq $file)) {
```

Comment:
I don't quite like this. It's hacky.

Reply:
I originally went for duplicating the entries in the above array holding the expected failures declarations, but that's a lot of duplication and maintenance overhead.

My other proposal (didn't go with it since it required more changes) was to allow for the user to define a regex in the `file` entry, and we could update all of those entries to have `(?:stream/)/test-suite.t`. But that'll still be a lot of duplication that seems unnecessary (since all future expected failures shared between the stream and http modules will also need to specify this regex).

What do you think?